src/web/js-interop.md

```diff
@@ -22,3 +22,77 @@ For help using the `js` package, see the following:
 [js-api]: {{site.pub-api}}/js
 [sass]: {{site.pub-pkg}}/sass
 
+## Future-proof JS interop
+
```

Comment:
We need a note here clearly stating that this work is experimental, similar to e.g. the note here: https://dart.dev/guides/libraries/objective-c-interop

Reply:
> {{site.alert.note}}
>       This interop feature is **experimental**,
>       and [in active development]().
> {{site.alert.end}}

What should I link "in active development" to? This issue: https://github.com/dart-lang/sdk/issues/50721 ?

Reply:
Maybe https://github.com/dart-lang/sdk/issues/35084?

Reply:
Oh wow, _there's_ the giant background issue I've been dreaming of! 😅 Thanks @parlough that'll be very helpful for all the upcoming JSinterop docs work